File: deepseek/1. Algoritmo Básico Ordenamiento por Burbuja/java/src/Burbuja.java
import java.util.Arrays;
import java.util.Scanner;

public class Burbuja {
    public static void Burbuja(int[] arr) {
        int n = arr.length;
        for (int i = 0; i < n-1; i++) {
            for (int j = 0; j < n-i-1; j++) {
                if (arr[j] > arr[j+1]) {
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese números separados por espacios: ");
        String[] input = scanner.nextLine().split(" ");
        int[] numbers = Arrays.stream(input).mapToInt(Integer::parseInt).toArray();
        Burbuja(numbers);
        System.out.println("Lista ordenada: " + Arrays.toString(numbers));
    }
}

File: deepseek/1. Algoritmo Básico Ordenamiento por Burbuja/C++/burbuja.cpp
#include <iostream>
#include <vector>
#include <sstream>
using namespace std;

void bubbleSort(vector<int>& arr) {
    int n = arr.size();
    for (int i = 0; i < n-1; i++) {
        for (int j = 0; j < n-i-1; j++) {
            if (arr[j] > arr[j+1]) {
                swap(arr[j], arr[j+1]);
            }
        }
    }
}

int main() {
    cout << "Ingrese números separados por espacios: ";
    string input;
    getline(cin, input);
    istringstream iss(input);
    vector<int> numbers;
    int num;
    while (iss >> num) {
        numbers.push_back(num);
    }
    bubbleSort(numbers);
    cout << "Lista ordenada: ";
    for (int num : numbers) {
        cout << num << " ";
    }
    cout << endl;
    return 0;
}